packages/sonarwhal/src/lib/cli/wizards/new-rule.ts

```diff
@@ -0,0 +1,421 @@
+        }];
+
+    if (!data.official) {
+        commonFiles.push({
+            destination: path.join(destination, '.sonarwhalrc'),
+            path: path.join(__dirname, SHARED_TEMPLATE_PATH, 'config.hbs')
+        });
+    }
+
+    const ruleFile = {
+        destination: path.join(destination, 'src'),
+        path: path.join(__dirname, TEMPLATE_PATH, 'rule.ts.hbs')
+    };
+    const testFile = {
+        destination: path.join(destination, 'tests'),
+        path: path.join(__dirname, TEMPLATE_PATH, 'tests.ts.hbs')
+    };
+
+    for (const file of commonFiles) {
+        const { destination: dest, path: p } = file;
+
+        const fileContent = await compileTemplate(p, data);
+
+        await mkdirpAsync(path.dirname(dest));
+        await writeFileAsync(dest, fileContent);
+    }
+
+    for (const rule of data.rules) {
+        const [ruleContent, testContent] = await Promise.all([compileTemplate(ruleFile.path, rule), compileTemplate(testFile.path, rule)]);
+
+        // e.g.: rule-ssllabs/src/ssllabs.ts
+        const rulePath = path.join(ruleFile.destination, `${rule.normalizedName}.ts`);
+        // e.g.: rule-ssllabs/tests/ssllabs.ts
+        const testPath = path.join(testFile.destination, `${rule.normalizedName}.ts`);
+
+        await Promise.all([mkdirpAsync(path.dirname(rulePath)), mkdirpAsync(path.dirname(testPath))]);
+
+        await Promise.all([writeFileAsync(rulePath, ruleContent), writeFileAsync(testPath, testContent)]);
+    }
+};
+
+/** Add a new rule. */
+export const newRule = async (actions: CLIOptions): Promise<boolean> => {
+    if (!actions.newRule) {
+        return false;
+    }
+
+    try {
+        const results = await inquirer.prompt(questions(QuestionsType.main));
+        const rules = [];
+
+        results.official = await isOfficial();
+
+        const askRules = async () => {
+            const rule = await inquirer.prompt(questions(QuestionsType.rule));
+
+            rules.push(rule);
+
+            if (rule.again) {
+                await askRules();
+            }
+        };
+
+        if (results.multi) {
+            await askRules();
+        }
+
+        results.rules = rules;
+
+        const rulePackage = new RulePackage(results);
+        const noOfficialOrigin: string = path.join(__dirname, 'no-official-files');
+        const files: string = path.join(__dirname, 'files');
+
+        if (!rulePackage.official) {
+            await copyFiles(noOfficialOrigin, rulePackage.destination);
+        }
+        await copyFiles(files, rulePackage.destination);
+        await generateRuleFiles(rulePackage.destination, rulePackage);
+
+        logger.log(`
+New ${rulePackage.isMulti ? 'package' : 'rule'} ${rulePackage.name} created in ${rulePackage.destination}
+
+--------------------------------------
+----          How to use          ----
+--------------------------------------
+1. Go to the folder rule-${rulePackage.normalizedName}
```

Comment:
Should we change this message if the rule is official? We should be using `yarn` in here